scripts/rebuild-lab.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Generally I'm reluctant to include platform specific scripts in repositories that are meant to be accessible for people across multiple platforms. Is there a reason why these need to be shell scripts, and not node / Python scripts? It makes sense for the .devcontainer since there the OS is controlled by the Dockerfile, but these generic ones are different.

Reply:
I moved these into the vscode folder and set them up as vscode tasks.